packages/utils/src/__tests__/WorkBookParser.test.js

```diff
@@ -58,7 +57,7 @@ describe('WorkBookParser', () => {
 
     it('should map the input to itself if no mapper is provided', async () => {
       const parser = new WorkBookParser();
-      await expect(parser.parse(WORK_BOOK)).resolves.toStrictEqual(SHEETS);
+      return expect(parser.parse(WORK_BOOK)).resolves.toStrictEqual(SHEETS);
```

Comment:
`await` would be also correct instead of `return` above - I used `return` just for consistency